docs/faq.md

```diff
@@ -80,3 +89,28 @@ Currently, the main benefit of [signed releases](/checks.md#signed-releases) is
 However, there are already moves to make it even more relevant. For example, the OpenSSF is working on [implementing signature verification for NPM packages](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/) which would allow a consumer to automatically verify if the package they are downloading was generated through a reliable builder and if it is correctly signed.
 
 Signing releases already has some relevance and it will soon offer even more security benefits for both consumers and maintainers.
+
+### Branch-Protection: How to setup a 10/10 branch protection on GitHub?
+
+To get a 10/10 score for Branch-Protection check using a non-admin token, you should have the following settings for your main branch:
+
+![image](/design/images/branch-protection-settings-non-admin-token.png)
```

Comment:
Ideally we would like to have alternative texts on those images, following the syntax `![alt text](/path/to/image)`.

However this is a difficult image to describe. In the blogposts, I've ben describing following this template, feel free to reuse:

[GitHub page where we define which protection rules will be applied to the selected branch, with the rules as checkboxes. Read below all the options ordered as they are displayed, followed the info if they're currently flagged or not: 
In the section named as "Protect matching branches", we have "Require a pull request before merging" flagged; "Require approvals" unflagged; "Dismiss stale pull request approvals when new commits are pushed" unflagged; "Require review from Code Owners" flagged; "Require approval of the most recent push" unflagged; "Require status checks to pass before merging" unflagged; "Require conversation resolution before merging" unflagged; "Require signed commits" unflagged; "Require linear history" unflagged; "Require deployments to succeed before merging" unflagged; "Lock branch" unflagged;  "Do not allow bypassing the above settings" flagged. In the section named as "Rules applied to everyone including administrators", we have "Allow force pushes" unflagged; "Allow deletions" unflagged]

Reply:
I think it can be simplified to

`[Branch-Protection page with the following options selected: "Require a pull request before merging", "Require approvals" with 2 approvers, "Require review from code owners", "Require status checks to pass before merging" with "Require branches to be up to date before merging" and at least one CI/CD check chosen. All other options are unchecked]`

Reply:
[ON HOLD] Analysing the best approach for the images...

Reply:
LGTM